src/main/java/core/dao/jdbc/DriverDaoJdbcImpl.java

```diff
@@ -102,8 +105,29 @@ private static Driver getDriver(ResultSet resultSet) throws SQLException {
         Long id = resultSet.getObject(1, Long.class);
         String name = resultSet.getString("name");
         String licenseNumber = resultSet.getString("license_number");
+        String login = resultSet.getString("login");
+        String password = resultSet.getString("password");
         Driver driver = new Driver(name, licenseNumber);
+        driver.setLogin(login);
+        driver.setPassword(password);
         driver.setId(id);
         return driver;
     }
+
+    @Override
+    public Optional<Driver> findByLogin(String login) {
+        String query = "SELECT * FROM drivers WHERE login = ? AND deleted = false";
```

Comment:
Don't use star here, in future this table could have more fields that would be redundant in your case